Aptoide/bds-sdk
# Appc 1146 sdk dialog wallet install to notification

**What does this PR do?**
- POA Notification
- Instead of going directly to the GooglePlay to Install Appcoins Wallet we first check if Aptoide is installed to install the wallet through there.

**Database changed?**
 No

**Where should the reviewer start?**
WalletUtils.java
PoaManager.java

**How should this be manually tested?**

This should be tested in Prod with no Wallet installed using 2 scenarios:

1- check if the Notification appears.
1a- Aptoide installed -> check if the notification appears and try to install the Wallet through and check if it goes to Aptoide.
1b-No Aptoide Installed -> check if the notification appears and try to install the Wallet through and check if it goes to Google Play.
2-reopen the application after the Wallet is installed to see if the notification appears.


  
**What are the relevant tickets?**
1146

**Questions:**

   No



**Code Review Checklist**
